expression/expression.go

```diff
@@ -243,17 +250,41 @@ func VecEvalBool(ctx sessionctx.Context, exprList CNFExprs, input *chunk.Chunk,
 			return nil, nil, err
 		}
 
+		j := 0
+		if eType == types.ETInt {
+			if err = expr.VecEvalInt(ctx, input, buf); err != nil {
+				return nil, nil, err
+			}
+
+			i64s := buf.Int64s()
+			for i := range sel {
+				if buf.IsNull(i) {
```

Comment:
Why don't we check `isEQCondFromIn` here?

Reply:
In order to be consistent with the original behavior. In the previous version, 'isEQCondFromIn' use [here](https://github.com/pingcap/tidb/blob/master/expression/expression.go#L191) where it needn't solve the int type. The int type is solved in [here](https://github.com/pingcap/tidb/blob/master/expression/chunk_executor.go#L373) where it ignores the 'isEQCondFromIn'